api/userservice.yaml

```diff
@@ -836,6 +836,23 @@ paths:
           description: CONFLICT - invalid/unavailable username
         500:
           description: INTERNAL SERVER ERROR - server encountered unexpected condition
+  /users/consultants/toggleWalkThrough:
```

Comment:
Could you please the existing "PATCH" endpoint of /users/data and just add there a new property for the walkthrough?

Reply:
You can then use all the existing service methods as well. And please add also tests for the new property

Reply:
done